src/level-ext.adoc

```diff
@@ -74,7 +74,7 @@ endif::[]
 11+| bit[0] - <<m_bit>> ({CAP_MODE_VALUE}-{cheri_cap_mode_name}, {INT_MODE_VALUE}-{cheri_int_mode_name})
 |Bits[4:3]| R | W | C | LM | EL | SL  | X | ASR | Mode^1^ |
 | 0-1   | ✔ | ✔ | ✔ | ✔  | ✔  | ∞   | ✔ |  ✔  | Mode^1^  | Execute + ASR (see <<infinite-cap>>)
-| 2-3   | ✔ |   | ✔ | ✔  | ✔  | ∞   | ✔ |     | Mode^1^  | Execute + Data & Cap RO
```

Comment:
This was meant to be inf.

Although it doesn't seem to matter what store-level a capability is if it cannot store, if we later want to introduce a  transitive store level policy this would be used to limit any capabilities loaded.

Getperm has to report something anyway, min and max being sensible defaults, and max is useful here.

Reply:
ok, fair enough